ci/travis/ci.sh

```diff
@@ -53,16 +61,33 @@ preload() {
   local variable_definitions
   variable_definitions=($(python "${ROOT_DIR}"/determine_tests_to_run.py))
   if [ 0 -lt "${#variable_definitions[@]}" ]; then
-    export "${variable_definitions[@]}"
+    local expression
+    expression="$(printf "%q " "${variable_definitions[@]}")"
+    eval "${expression}"
+    printf "%s\n" "${expression}" >> ~/.bashrc
```

Comment:
what does this section do?

Reply:
It emits the assignments (`RAY_CI_XYZ=1`) into the `.bashrc`. I could keep them as exports, but I'm trying to architecture it so that we don't try to refer to them from external scripts later, since those variables are CI-specific and shouldn't be depended on by scripts that could conceivably be called without these variables being defined. Let's make sure the tests are working before we merge this though!